docs/using-mina/how-to-use-zkapp.mdx

```diff
@@ -2,4 +2,39 @@
 title: How to use a zkApp
 ---
 
-Describe
+import Subhead from '@site/src/components/common/Subhead';
+
+# Install a Wallet
+
+<Subhead>Learn how to interact with a zero-knowledge smart contract</Subhead>
+
+:::info
+
+Please note that zkApp programmability is not yet available on Mina Mainnet, but zkApps can now be deployed to Berkeley Testnet.
+
+:::
+
+### Prerequisites
+
+To use a zkApp:
+1. Install a Chrome zkApp-compatible Mina wallet.
+2. Make sure that your Mina wallet contains MINA to pay for transaction fees. 
+
+:::note
+
+The Mina community has created a variety of different wallets. Only the [Auro Wallet for Chrome](https://www.aurowallet.com) supports interactions with zkApps.
+
+:::
+
+### Instructions
+
+1. Visit the zkApp in a web browser. For example,  mycoolzkapp.com.
+2. Interact with the zkApp as intended. For example, make a move in a game, enter in your age, and so on.
+3. Click the confirmation button to send the transaction to the Mina network.
+4. In your Mina browser wallet extension, confirm the transaction again.
```

Comment:
-again 

They're only confirming it once. The button on the page starts that interactions, but they're actually confirming via the wallet.